src/Sulu/Bundle/SecurityBundle/Command/CreateUserCommand.php

```diff
@@ -216,7 +136,7 @@ protected function interact(InputInterface $input, OutputInterface $output)
         if (!$input->getArgument('username')) {
             $question = new Question('Please choose a username: ');
             $question->setValidator(
-                function($username) use ($userRepository) {
+                static function($username) use ($userRepository) {
```

Comment:
I think instaed of making these functions `static`, I would omit the `use` and write `$this->userRepository`. Have not heard about static anonymous functions before, so that might confuse some people.